uvtools/dspec.py

```diff
@@ -109,7 +109,7 @@ def calc_width(filter_size, real_delta, nsamples):
     return (uthresh, lthresh)
 
 def fourier_filter(x, data, wgts, filter_centers, filter_half_widths, suppression_factors,
-                   mode, filter2d, fitting_options, cache=None, filter_dim=1, skip_wgt=0.1,
+                   mode, filter2d, fitting_options=None, cache=None, filter_dim=1, skip_wgt=0.1,
```

Comment:
I'm going to leave this as None since it's sometimes bad practice to use a mutable in the default kwarg (it ends up changing the function signature, for example, that shows up when you call up the documentation in a notebook so you'll see 'fitting_options={key1:item1, key2:item2...}') in the docstring.